UsageRuleTUISample/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.iml
+.gradle
+local.properties
+.idea/**
```

Comment:
You don't have to change this one, but I think it is overkill. The "**" means all files and directories, just so you're aware. That same thing is true of the other directories we list.

Reply:
Never messed with this file, not sure why they would hae changed.  Anyway, went ahead and made the changes, pushed another commit.